app/src/App.tsx

```diff
@@ -1,22 +1,71 @@
 import { ThemeProvider } from '@emotion/react';
 import appTheme from 'themes/appTheme';
-import { Button, Card, Typography } from '@mui/material';
 import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Header from './UIComponents/Header';
+import Home from './PageComponents/Home';
+import Login from './PageComponents/Login';
+import Dashboard from './PageComponents/Dashboard';
+import Survey from './PageComponents/Survey';
+import Schedule from './PageComponents/Schedule';
+import ScheduleManage from './PageComponents/ScheduleManage';
+import ScheduleGenerate from './PageComponents/ScheduleGenerate';
+import ScheduleTimetable from './PageComponents/ScheduleTimetable';
+import ProfessorProfile from './PageComponents/ProfessorProfile';
 
 function App() {
   return (
     <ThemeProvider theme={appTheme}>
-      {/* This is our main app tree. All nested components in this tree will inherit properties from those above it. Our router should be the root of this tree. */}
-      <Card>
-        {/* This card can go away when we have a router */}
-        <Typography variant="h4">Hello from a MUI card!</Typography>
-        <Button color="primary" variant="contained">
-          Primary Color Button
-        </Button>
-        <Button color="secondary" variant="contained">
-          Secondary Color Button
-        </Button>
-      </Card>
+      <Router>
```

Comment:
Let's put this in a separate component called Router.tsx so our App.tsx stays lean. Root directory of app/ should be good.